### ISSUE TITLE
Add Custom Main Menu

### ISSUE THREAD

**Issue by Nincodedo** [Nincraft/NincraftElectricBoogaloo2RevengeoftheFallen#140]
http://minecraft.curseforge.com/mc-mods/226406-custom-main-menu

And make it fancy.

**Comment by Nincodedo** [comment]
Not really necessary for 1.5 beta so moved to 1.0 release.

**Comment by UndeadZeratul** [comment]
Agreed, we don't necessarily need it until we actually start sending it out to people.
